test/tests/functional/pbs_hook_execjob_prologue.py

```diff
@@ -247,12 +250,21 @@ def test_prologue_hook_does_not_execute_twice_on_pbsdsh(self):
         attr = {'event': 'execjob_prologue'}
         self.server.create_import_hook(hook_name, attr, hook_body)
 
-        j = Job(TEST_USER, {'Resource_List.select': '2:ncpus=1',
-                            'Resource_List.place': 'scatter'})
-        j.create_script('pbsdsh -n 1 hostname')
+        j = Job(PBSROOT_USER, {'Resource_List.select': '2:ncpus=1',
```

Comment:
Why did you change this from TEST_USER to PBSROOT_USER?

Reply:
@bhroam I changed this because the job o/p file will then be accessible to pbsroot easily in the test case.

Reply:
I think you are making an assumption this test will always be run by pbsroot.  This may be the case by internal Altair automated systems, but the test should be able to be run by any user.

Reply:
Yes, you are right. I've changed the user back to TEST_USER.